py/client-ticking/src/pydeephaven_ticking/util.py

```diff
@@ -13,6 +13,12 @@ def _to_sequence(v: Union[T, Sequence[T]] = None) -> Sequence[T]:
     """This  enables a function to provide parameters that can accept both singular and plural values of the same type
     for the convenience of the users, e.g. both x= "abc" and x = ["abc"] are valid arguments.
     (adapted from table_listener.py)
+
+    Args:
+        v (Union[T, Sequence[T]]) : The input argument
+
+    Returns:
+        A sequence containing 0, 1, or multiple values
```

Comment:
maybe we should  want to make the module 'private' ? i..e. rename it to `_util`.

Reply:
Changed, thank you